<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNewSnapshot</class>
 <widget class="QDialog" name="dlgNewSnapshot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Snapshot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>New Snapshot</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Config</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbConfigs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Snapshot name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lneName">
        <property name="placeholderText">
         <string>Type a short description about your snapshot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="optSingle">
          <property name="text">
           <string>Single</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="optPre">
          <property name="text">
           <string>Pre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="optPost">
          <property name="text">
           <string>Pos&amp;t</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPaired">
          <property name="text">
           <string>Paired with :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbPaired">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Cleanup Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmbCleanAlgo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Timeline</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Empty-pre-post</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Command :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QTextBrowser" name="txtCommandDisplay"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgNewSnapshot</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgNewSnapshot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optPost</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbPaired</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optPost</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblPaired</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
